contracts/base/ERC4337Account.sol

```diff
@@ -51,6 +51,7 @@ abstract contract ERC4337Account is IERC4337Account {
 
     function addDeposit() public payable virtual {
         IEntryPoint(entryPoint()).depositTo{ value: msg.value }(address(this));
+        emit DepositAdded(address(this), msg.sender, msg.value);
```

Comment:
can change to some other name

besides entrypoint already emits this event
emit Deposited(account, newDeposit);

Reply:
ok I'll remove it, if not necessary